cryptoapis/test/balance.test.ts

```diff
@@ -0,0 +1,213 @@
+import { assert } from 'chai'
```

Comment:
Again, we should reuse this behavior so we don't duplicate so much code.

Reply:
Let's make a new test package for this `@chainlink/adapter-test-helpers`, where we'll add a behavior section and define different behaviors that we can reuse across different adapters. Starting with these PoR adapters. Let's make a few behaviors like `shouldBehaveLikeBlockchainIndexer` where we could test for validation errors on input sequences and then`shouldBehaveLikeBitcoinIndexer` where we can do integration tests for all PoR adapters.